fvm/crypto.go

```diff
@@ -61,34 +54,78 @@ func newHasher(hashAlgo hash.HashingAlgorithm) hash.Hasher {
 		return hash.NewSHA2_256()
 	case hash.SHA3_256:
 		return hash.NewSHA3_256()
+	case hash.SHA2_384:
+		return hash.NewSHA2_384()
+	case hash.SHA3_384:
+		return hash.NewSHA3_384()
```

Comment:
What is the reason for leaving them out? We added them to Cadence, so I thought we could just implement them?

Reply:
I am thinking of deprecating them from the crypto library since they are not used anywhere (I'll use them if Cadence needs them). They are also one for the reasons the type `crypto.Hash` is a slice and not a constant-length array. Their usage is rather rare in applications outside Flow too. 

The reason why they existed in the first place is that BLS signatures used them in the first implementation, before switching to KMAC128.

Reply:
We can leave them if they were left in Cadence for now.

Reply:
I actually thought they were removed from Cadence after the comment [here](https://github.com/onflow/cadence/pull/633#discussion_r589692799).

Reply:
Soon. we will need a KMAC for BLS, so we will have a non-SHA2/3-256 hasher for sure. 
If the work has been done already, we can keep them. I just want to make sure they remain Cadence tools and won't interfere with flow accounts.

Reply:
However, BLS could be removed from Cadence for now if you confirm [it's not usable](https://github.com/onflow/flow-go/pull/529#discussion_r595652120).